packages/core/src/components/text-field/_text-field.scss

```
@@ -169,4 +169,10 @@
       }
     }
   }
+
+  [rows].#{$ray-class-prefix}text-area {
+    &__input {
+      height: auto !important;
```

```suggestion
      height: auto; // ie11 fallback
      height: unset;
```

`auto` value prevents the user from resizing. Let's use unset and leave auto as a fallback for ie.